pyfr/integrators/dual/pseudo/pseudosteppers.py

```diff
@@ -337,3 +340,66 @@ class DualRK45PseudoStepper(DualRKVdH2RPseudoStepper):
     @property
     def _pseudo_stepper_order(self):
         return 4
+
+
+class DualDenseRKPseudoStepper(BaseDualPseudoStepper):
+    pseudo_stepper_name = 'dense'
```

Comment:
Not sure about this.  We might want to simply synthesise classes on the fly.  So as part of the init process we can list all of the dense schemes in PyFR and then generate (using the type function) suitable subclasses.  Everything else then works as expected with no changes needed to inifiles or scheme-selection logic.